stream-applications-release-train/stream-applications-docs/src/main/asciidoc/index-docinfo.xml

```diff
@@ -1,7 +1,7 @@
 <productname>Spring Cloud Data Flow</productname>
 <releaseinfo>{project-version}</releaseinfo>
 <copyright>
-	<year>2013-2021</year>
+	<year>2013-2022</year>
 	<holder>Pivotal Software, Inc.</holder>
```

Comment:
As well as this one.

Not sure though, that this file is used anywhere...

@sobychacko ?

Reply:
I updated this to be `VMware, Inc` as I am not sure if this is used anywhere or not - so just made it accurate.

Reply:
I think this file is a remnant from some previous times. We can probably safely delete this for the 2022 release.